airflow/utils/db.py

```diff
@@ -336,6 +336,8 @@ def resetdb():
     Clear out the database
     """
     from airflow import models
+    # noinspection PyUnresolvedReferences
+    from airflow.models.serialized_dag import SerializedDagModel  # noqa: F401 isort # isort:skip
```

Comment:
Should we just sort it? Do we want to import anything in `airflow.models.serialized_dag` when store_serialized_dag=False ?

Reply:
Yeah. It is copy paste, really.

Reply:
And yes - we want to keep it here because the DB table should be created always - no matter what the value of "store_serialized_dag` is.

Reply:
This code is needed to delete serialised_dag table when resetdb is performed.

Reply:
Table would be created by Alembic not the Model even when using resetdb

Reply:
Not really. Try it yourself :). If you remove the import here and run 'db init' -> 'db reset' sequence you will get an error that the table already exists. Resetdb drops all the tables for which model is registered. Since we removed SerializedDagModel from models.__init__.py (we had to because it caused cyclic imports) we had to add it manually heere so that db reset deletes the table.

Reply:
aah ok, my bad in that case :)

Reply:
Since we removed it from models.__init__.py